### ISSUE TITLE
Content of the tabs on the "Contact Us" page is not read by the screen reader when using up and down arrows (as is the case with other pages)

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#232]
Only right and left arrow should be used to shift between the tabs. Down arrow should lead to an active panel (starting from either selected or not selected tab).

Noticed with NVDA on Windows.
